pkg/v1/cli/command/core/config.go

```diff
@@ -182,9 +182,14 @@ func setEdition(cfg *configv1alpha1.ClientConfig, edition string) error {
 	editionOption := configv1alpha1.EditionSelector(edition)
 
 	switch editionOption {
-	case configv1alpha1.EditionCommunity,
-		configv1alpha1.EditionStandard:
+	case configv1alpha1.EditionCommunity, configv1alpha1.EditionStandard:
 		cfg.SetEditionSelector(editionOption)
+		// when community edition is set, configuration the compatibility file to use
```

Comment:
I think this comment needs to be updated; `configuration` -> `configure`, maybe?

Reply:
yes,typo, fixed!